workers/unity/Packages/io.improbable.gdk.tools/GdkToolsConfigurationWindow.cs

```diff
@@ -160,5 +164,26 @@ private void DrawCodeGenerationOptions()
 
             EditorGUIUtility.labelWidth = previousWidth;
         }
+
+        private void DrawCustomSnapshotDir()
+        {
+            GUILayout.Label(CustomSnapshotPathLabel, EditorStyles.boldLabel);
+            GUILayout.Space(EditorGUIUtility.standardVerticalSpacing);
+
+            using (new EditorGUILayout.HorizontalScope())
+            {
+                EditorGUI.BeginDisabledGroup(true);
+                EditorGUILayout.TextField(CustomSnapshotPathLabel, toolsConfig.CustomSnapshotPath);
+                EditorGUI.EndDisabledGroup();
+                if (GUILayout.Button("Open", EditorStyles.miniButtonRight, GUILayout.ExpandWidth(false)))
+                {
+                    var path = EditorUtility.OpenFilePanel("Select snapshot", Path.GetFullPath(toolsConfig.CustomSnapshotPath), "snapshot");
```

Comment:
Should this be `Path.GetDirectoryName()` rather than `Path.GetFullPath()`? 

If we want the directory that the currently selected snapshot is in. I'm not 100% clear on the behaviour of `EditorUtility.OpenFilePanel()`